src/AppBundle/Entity/Post.php

```diff
@@ -7,7 +7,7 @@
 /**
  * Post
  *
- * @ORM\Table(name="posts")
+ * @ORM\Table(name="posts1")
```

Comment:
Is Post entity From appBundle used at all? it seems the same entity is at src/DemoBundle/Entity/Post.php